tests/test_features/test_manifold.py

```diff
@@ -314,3 +314,13 @@ def test_manifold_pandas(self):
         oz.cbar.set_ticks([])
         # TODO: find a way to decrease this tolerance
         self.assert_images_similar(oz, tol=40)
+
+    def test_manifold_quick_method(self):
+        """
+        Test Manifold quick method with colors being set.
```

Comment:
Can you update this docstring to something like `Test Manifold quick method with single target.`